tiledb/sm/global_state/tbb_state.cc

```diff
@@ -99,25 +117,39 @@ Status init_tbb(const Config* config) {
   return Status::Ok();
 }
 
-}  // namespace global_state
-}  // namespace sm
-}  // namespace tiledb
-
 #else
 
-namespace tiledb {
-namespace sm {
-namespace global_state {
-
-int tbb_nthreads_ = 0;
+/** The ThreadPool to use when TBB is disabled. */
+std::shared_ptr<ThreadPool> global_tp_;
 
 Status init_tbb(const Config* config) {
```

Comment:
Change function name?

Reply:
Leaving as-is based on previous discussion. To be changed in the future.